src/richie/apps/courses/models/organization.py

```diff
@@ -98,9 +98,12 @@ class OrganizationPluginModel(CMSPlugin):
     Organization plugin model handles the relation from OrganizationPlugin
     to their Organization instance
     """
-
-    organization = models.ForeignKey(Organization)
-
+    organization = models.ForeignKey(
+            "Organization",
+            related_name="organization",
+            limit_choices_to={"extended_object__publisher_is_draft": True},
```

Comment:
@sveetch in the description of your PR you wrote "only publised" and her you are filtering "only draft". The description is wrong and this is good ;-)

Reply:
Well, that was a mistake. I wanted to make sure i showed only one version.
Should it be "extended_object__publisher_is_draft": False
if i want to show published organization only ? Which one would be better ?

Reply:
No you should leave it like this. We only edit draft versions. The public version is set when you publish and DjangoCMS copies the draft to the published version.

Reply:
allright 👍 Thanks a lot.